Radiant/fileWatcher.js

```diff
@@ -98,21 +98,17 @@ const checkM3U8 = (file) => {
  */
 const checkFile = function (info, previousSize){
     const ext = info.path.replace(/^.*[\\\/]/, '').split('.')[1];
-    if(ext === 'm3u8'){
-        uploadFile(info, false);
-    } else {
-        fs.stat(info.path, (err, fileInfo) => {
-            if(err === null) {
-                if(fileInfo.size === previousSize && fileInfo.size > 0) {
-                    uploadFile(info, false);
-                } else {
-                    streamTracker[info.path].throttleCheck(info, fileInfo.size);
-                }
-            }  else {
-                Logger.error(`File not found ${err}`);
+    fs.stat(info.path, (err, fileInfo) => {
+        if(err === null) {
+            if(ext !== 'm3u8' && fileInfo.size === previousSize && fileInfo.size > 0) {
+                uploadFile(info, false);
+            } else {
+                streamTracker[info.path].throttleCheck(info, fileInfo.size);
             }
```

Comment:
does the m3u8 get uploaded somewhere else now?

Reply:
yes it uploads after the ts file is uploaded :)